docs/resources/workflow.md

```diff
@@ -87,7 +87,7 @@ resource "incident_workflow" "autoassign_incident_lead" {
 - `include_private_incidents` (Boolean) Whether to include private incidents
 - `name` (String) Name provided by the user when creating the workflow
 - `once_for` (List of String) This workflow will run 'once for' a list of references
-- `runs_on_incident_modes` (List of String) Incidents in these modes will be affected by the workflow
+- `runs_on_incident_modes` (Set of String) Incidents in these modes will be affected by the workflow
```

Comment:
do you want to add the unreleased changes in the changelog in this PR, for traceability?

Reply:
Ooh yeah, how should I go about doing that?